pkg/controllers/job/job_controller_actions.go

```diff
@@ -296,14 +296,19 @@ func (cc *jobcontroller) syncJob(jobInfo *apis.JobInfo, updateStatus state.Updat
 	}
 
 	var jobCondition batch.JobCondition
+	oldStatus := job.Status
 	if !syncTask {
 		if updateStatus != nil {
-			if updateStatus(&job.Status) {
-				job.Status.State.LastTransitionTime = metav1.Now()
-				jobCondition = newCondition(job.Status.State.Phase, &job.Status.State.LastTransitionTime)
-				job.Status.Conditions = append(job.Status.Conditions, jobCondition)
-			}
+			updateStatus(&job.Status)
+		}
+
+		if equality.Semantic.DeepEqual(job.Status, oldStatus) {
+			klog.V(3).Infof("Job <%s/%s> has not updated for no changing", job.Namespace, job.Name)
```

Comment:
I think log level 4 is better cause it's not a vital info.

Reply:
changes done